lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -112,18 +114,6 @@ def group_name
       type.downcase.pluralize
     end
 
-    def run_kubectl(*args)
-      raise FatalDeploymentError, "Namespace missing for namespaced command" if namespace.blank?
-      raise FatalDeploymentError, "Explicit context is required to run this command" if context.blank?
```

Comment:
We need to keep these two lines and do a `super` passing the namespace and context in here. Right now all the invocations by these resources would fail because the kwargs are missing.